website/content/assets/images/README.md

```diff
@@ -0,0 +1,23 @@
+# Images
+
+## background.jpg
+
+Public domain image sourced from [NASA]().
```

Comment:
URL?

Is this even right? Pretty sure this image is from the design agency.

Reply:
Right. These test images shouldn't be here. Dropped. I've added proper links for the third-party assets in assets/README.md.